src/anurag/Shop.java

```diff
@@ -0,0 +1,70 @@
+package anurag;
+
+public class Shop {
+
+	int maggie = 50;
+	int dosa = 43;
+	int oil = 39;
+	int panipuri = 43;
+	int masala = 73;
+
+	void calculate(String item, int qty) { // qty is the quantity purchased by the customer.
+
+		if (item == "Maggie" && qty <= maggie) {
+			maggie = maggie - qty;
+			// System.out.println("remaining Item of maggie is" + maggie);
+		} else if (item == "Dosa" && qty <= dosa) {
+			dosa = dosa - qty;
+			// System.out.println("remaining Item of dosa is " + dosa);
+		} else if (item == "Oil" && qty <= oil) {
+			oil = oil - qty;
+			// System.out.println("remaining Item of oil is " + oil);
+		} else if (item == "PaniPuri" && qty <= panipuri) {
+			panipuri = panipuri - qty;
+			// System.out.println("remaining Item of oil is " + panipuri);
+		} else if (item == "Masala" && qty <= masala) {
+			masala = masala - qty;
+			// System.out.println("remaining Item of masala is " + masala);
+		}
+
+	}
+
+	void OutOfStockItems() {
+		if (maggie <= 0)
```

Comment:
stock never be in minus, logically looks wrong!

Do not change in this file, but improve thinking on such scenarios in next PR.